### PR TITLE
Clarify synthetics params / secrets docs

### PR DESCRIPTION
**Pull request by andrewvc**
Clarify and add detail to section on synthetics security / working with sensitive values

### CODE REVIEW

File: docs/en/observability/synthetics-params-secrets.asciidoc [hunk 1]
```diff
@@ -148,11 +150,15 @@ Your synthetics scripts may require the use of passwords or other sensitive secr
...
-Because synthetics scripts have no limitations, a malicious script author could write a
-synthetics journey that exfiltrates `params` and other data at runtime.
+Please note that params are viewable in plain-text by administrators and other users with "all" privileges for
+the synthetics app.
+Additionally, note that synthetics scripts have no limitations on accessing these values, and  a malicious script author could write a
+synthetics journey that exfiltrates `params` and other data at runtime. 
 Do *not* to use truly sensitive passwords (for example, an admin password or a real credit card)
 in *any* synthetics tools.
 Instead, set up limited demo accounts, or fake credit cards with limited functionality.
+If you want to limit access to parameters ensure that that users who are not supposed to access those values do not have "all" privileges
```
Comment (vigneshshanmugam) [inline docs/en/observability/synthetics-params-secrets.asciidoc:160]:
```suggestion
If you want to limit access to parameters ensure that users who are not supposed to access those values have "read" privileges
```

**Review by vigneshshanmugam — APPROVED** [review]

File: docs/en/observability/synthetics-params-secrets.asciidoc [hunk 2]
```diff
@@ -2,10 +2,12 @@
 [[synthetics-params-secrets]]
 = Work with params and secrets
 
-Params allow you to use dynamically defined values, including sensitive information, in your
+Params allow you to use dynamically defined values, in your
 synthetic monitors. For example, you may want to test a production website with a particular
 demo account whose password is only known to the team managing the synthetic monitors.
 
+Please read the <<synthetics-sensitive-secret, documentation on sensitive values>>for more information on security-sensitive use cases.
```
Comment (vigneshshanmugam) [inline docs/en/observability/synthetics-params-secrets.asciidoc:9]:
```suggestion
Please read the <<synthetics-secrets-sensitive, documentation on sensitive values>>for more information on security-sensitive use cases.
```

File: docs/en/observability/synthetics-params-secrets.asciidoc (hunk 1 above)
Comment (andrewvc) [reply docs/en/observability/synthetics-params-secrets.asciidoc:160]: I'm worried that this isn't quite right in that users with `read` privileges can still view a lot of network request metadata including request headers that might contain credentials. Do you agree we need to document / caveat this? I generally think that we should advise folks to think of it as more binary, if you can view the monitor you may be able to view credentials. Even if that isn't true all of the time, it is some of the time.

Your thoughts?

Comment (vigneshshanmugam) [reply docs/en/observability/synthetics-params-secrets.asciidoc:160]: Makes sense. Lets keep it the way you proposed.

File: docs/en/observability/synthetics-params-secrets.asciidoc [hunk 3]
```diff
@@ -2,10 +2,12 @@
 [[synthetics-params-secrets]]
 = Work with params and secrets
 
-Params allow you to use dynamically defined values, including sensitive information, in your
+Params allow you to use dynamically defined values, in your
```
Comment (dedemorton) [inline docs/en/observability/synthetics-params-secrets.asciidoc:5]:
```suggestion
Params allow you to use dynamically defined values in your
```

**Review by dedemorton — CHANGES_REQUESTED** [review]
Just a few editorial changes.

BTW, it's much better if you submit your changes against the main branch (rather than a versioned branch) so that we can easily backport the changes. To do this, you just need to switch to master before you click the Edit link in the docs.

**Comment by vigneshshanmugam** [comment]
@dedemorton Thanks for the review, Andrew is out of office, I will open a new PR to address your review comments and ask for final review.

**Comment by vigneshshanmugam** [comment]
Closed in favor of https://github.com/elastic/observability-docs/pull/3691

**Pull request closed without merging**